anesthetic/samples.py

```diff
@@ -206,33 +206,19 @@ def plot_2d(self, axes, *args, **kwargs):
         types: dict, optional
             What type (or types) of plots to produce. Takes the keys 'diagonal'
             for the 1D plots and 'lower' and 'upper' for the 2D plots.
-            The options for 'diagonal are either:
+            The options for 'diagonal are:
                 - 'kde'
                 - 'hist.
-            The options for 'lower' and 'upper' are either:
+            The options for 'lower' and 'upper' are:
                 - 'kde'
                 - 'scatter'
             Default: {'diagonal': 'kde', 'lower': 'kde'}
 
-        diagonal_kwargs: dict, optional
-            kwargs only for the diagonal (1D) plots. This is useful when there
-            is a conflict of kwargs for different types of plots.
-            Note that any kwargs directly passed to plot_2d will overwrite any
-            kwarg with the same key passed to diagonal_kwargs.
-            Default: {}
-
-        lower_kwargs: dict, optional
-            kwargs only for the lower 2D plots. This is useful when there
-            is a conflict of kwargs for different types of plots.
-            Note that any kwargs directly passed to plot_2d will overwrite any
-            kwarg with the same key passed to lower_kwargs.
-            Default: {}
-
-        upper_kwargs: dict, optional
-            kwargs only for the upper 2D plots. This is useful when there
-            is a conflict of kwargs for different types of plots.
-            Note that any kwargs directly passed to plot_2d will overwrite any
-            kwarg with the same key passed to upper_kwargs.
+        diagonal_kwargs, lower_kwargs, upper_kwargs: dict, optional
+            kwargs for the diagonal (1D)/lower or upper (2D) plots. This is
+            useful when there is a conflict of kwargs for different types of
+            plots.  Note that any kwargs directly passed to plot_2d will
+            overwrite any kwarg with the same key passed to diagonal_kwargs.
```

Comment:
Nicely reduces the repetition. Last reference to `diagonal_kwargs` should be changed to `sub-kwargs` or a mention of all of them, though.

Reply:
Good spot